modules/handlers/anonym_comment.py

```diff
@@ -16,5 +20,42 @@ def anonymous_comment_msg(update: Update, context: CallbackContext):
     info = EventInfo.from_message(update, context)
 
     if info.chat_id == config_map['meme']['channel_group_id'] and info.message.via_bot is None:
-        info.message.copy(chat_id=info.chat_id, reply_to_message_id=info.message.reply_to_message.message_id)
+        message = info.message
+        poll = message.poll  # if the message is a poll, get its reference
+        anon_credits = 'by: ' + choice(read_md("anonym_names").split("\n"))
+        try:
+            if poll or message.sticker:  # makes sure the poll is anonym
+              if poll:
+                sent_message_id = info.bot.send_poll(chat_id=info.chat_id,
+                                                  question=f'{poll.question}\n\n{anon_credits}',
+                                                  options=[option.text for option in poll.options],
+                                                  type=poll.type,
+                                                  allows_multiple_answers=poll.allows_multiple_answers,
+                                                  reply_to_message_id=info.message.reply_to_message.message_id if info.message.reply_to_message else None,
+                                                  correct_option_id=poll.correct_option_id).message_id
+              else:
+                sent_message_id = info.bot.send_sticker(chat_id=info.chat_id,
+                                                      sticker=message.sticker.file_id,
+                                                      reply_to_message_id=info.message.reply_to_message.message_id if info.message.reply_to_message else None).message_id
+
+              info.bot.send_message(chat_id=info.chat_id,
+                                    text=anon_credits,
+                                    reply_to_message_id=sent_message_id)
+          
+            elif message.text and message.entities:  # mantains the previews, if present
+                info.bot.send_message(chat_id=info.chat_id,
+                                      text=message.text + f'\n\n{anon_credits}',
+                                      entities=message.entities,
+                                      reply_to_message_id=info.message.reply_to_message.message_id if info.message.reply_to_message else None)
+            else:
+                info.bot.copy_message(chat_id=info.chat_id,
+                                      from_chat_id=message.chat_id,
+                                      message_id=message.message_id,
+																			caption=f'{message.caption}\n\n{anon_credits}' if message.caption else anon_credits,
+                                      reply_to_message_id=info.message.reply_to_message.message_id if info.message.reply_to_message else None)
+																			
+        except (BadRequest) as e:
+            Logger.error("Sending the post on send_post_to: %s", e)
+            return False
```

Comment:
Just return, the linter complains that there isn't supposed to be any return value
```diff
- return False
+ return
```